app/views/users/new.html.erb

```diff
@@ -1,3 +1,23 @@
-<% provide :title, "Sign up" %>
-<h1>Sign up</h1>
-<p>This will be a signup page for new users.</p>
+<% provide :title, t("signup.title") %>
+<h1><%= t("signup.title") %></h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(@user) do |f| %>
+      <%= render "shared/error_messages" %>
+
+      <%= f.label :name, t("signup.name") %>
```

Comment:
```suggestion
      <%= f.label :name, t "signup.name" %>
```
Sau t ko còn j thì em bỏ ngoặc dc

ngoài ra, i18n có support theo model ấy. nó sẽ tự động I18n ko cần gọi thế này, em có thể tham khảo nhé 

https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models